### PR TITLE
Remove unused dependency

### PR DESCRIPTION
**Pull request by anton-efimenko**
The `expiringdict` dependency is not necessary.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
           'cryptography>=0.5',
           'cython>=0.21.1',
           'dnsq>=1.1.6',
-          'expiringdict>=1.1.2',
```
Comment (horkhe) [inline setup.py]: Well, it is a transitive dependency - flanker->dnsq->expiringdict, but you are right it is not needed here.

Comment (anton-efimenko) [inline setup.py]: Thanks! This prevents people from installing flanker.

Comment (horkhe) [inline setup.py]: That still will be the case because expiringdict is a transitive dependency.

**Comment by horkhe** [comment]
@anton-efimenko #164 should actually fix the problem

**Pull request merged**